.github/workflows/build.yml

```diff
@@ -34,7 +45,7 @@ jobs:
 
       - name: Run Cake script
         id: build-cake
-        uses: cake-build/cake-action@v1
+        uses: cake-build/cake-action@master
```

Comment:
Is there a reason that we can't pin to the version/tag of the action?

Reply:
That will be the next question :D which release will this get into?

Reply:
@dellis1972  This PR will be released as part of 5.0 release, planned shortly (as son as we get the cycles to do so) after .NET 9 release tomorrow.

You could already now try the pre-release via our Azure Artifacts feed.
https://dev.azure.com/cake-build/Cake/_artifacts/feed/cake/NuGet/Cake.Frosting/overview/5.0.0-alpha0030
```xml
  <PackageReference Include="Cake.Frosting" Version="5.0.0-alpha0030" />
```

if you want to test in your PR now, you can add the public source as
```xml
<add key="cake" value="https://pkgs.dev.azure.com/cake-build/Cake/_packaging/cake/nuget/v3/index.json" />
```